pool/app/experiment/experiment.ml

```diff
@@ -1,12 +1,28 @@
 include Entity
 include Event
 module Guard = Entity_guard
-module Repo = Repo
+
+module Repo = struct
+  module Public = struct
```

Comment:
How about adding `module Public = Repo_public` to the Repo file directly?

Reply:
That results in a dependency cycle.